tests/generic/nft_trades_column_values_matching_test.sql

```diff
@@ -15,7 +15,7 @@
         model_sample.{{column_name}} as model_{{column_name}} {% if not loop.last %},{% endif %}
         {% endfor -%}
         from {{seed_file}} seed
-        left join (
+        inner join (
```

Comment:
@jeff-dude inner join is required for aggregated seed file like dex_trade_seeds for this generic test framework. Otherwise we may need to store seeds in different files for different model.

Also left join is applicable when all seeds are supposed to be available in the view, like dex_trade_seed to dex.trades meta-table.

Reply:
(Not solved yet)

Reply:
yes, the intention is to move the test to parent ```dex.trades``` model, so all projects would be included and left join should be fine. as you mentioned, its still WIP to remove tests from all models and apply at parent.